debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -27,6 +27,8 @@
  */
 public abstract class RelationalDatabaseConnectorConfig extends CommonConnectorConfig {
 
+    private static final int DEFAULT_ROWS_FETCH_SIZE = 10240;
```

Comment:
This has been taken care of already as part of https://github.com/debezium/debezium/pull/855. The code here won't be needed any longer, it should become apparent during the rebase.